### DISCUSSION TITLE
비글챌 규칙(2024-02-21 업데이트)

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#7] [가이드라인]
### 슬랙 활용법
- 상시 알람을 받을 수 있도록 사이드 바의 `환경설정>알림`에서 알람 설정을 변경해주세요!
<div align=center><img src="https://github.com/amaran-th/every-week-posting-challenge/assets/81465068/42bf3101-d7cd-48d1-9a8e-a92884b05bdf" width=400/></div>

- 이모티콘은 자유롭게 추가하셔도 됩니다~

### 챌린지 규칙

- 주 1회 최소 1개 이상의 글을 업로드하는 것을 기본 규칙으로 합니다.

- 여러 개의 글을 업로드할 경우 디스커션도 여러 개 올려주셔야 합니다!

- 한 주의 단위는 월요일 00:00부터 일요일 24:00(자정)까지 입니다.

- 글은 디스커션에 직접 작성하셔도 되고, 다른 포털(블로그, 노션 등)에 작성한 글의 링크를 첨부하여 올려도 괜찮습니다! 당연하지만 해당 주가 아닌 다른 시기에 작성된 글은 인정되지 않습니다.
- 글을 한 주동안 완성하지 못한 경우 작성중인 글의 초안을 게시하는 것도 글 작성으로 인정됩니다.

- 만약 한 주 동안 글을 작성하지 않았을 경우, 그 다음 주까지 200자 이내의 반성문을 작성해서 디스커션 반성문 카테고리에 올려주셔야 합니다. 반성문을 작성하지 않을 경우 벌금(5000원)이 부과됩니다.
- 벌금은 BDD 운영 자금으로 사용됩니다.
- 벌금 납부 : 토스뱅크 1000-7769-0601 송세연

- 라벨은 자유롭게 추가하셔도 됩니다! 라벨 추가 후엔 슬랙 잡담방에 알려주세요!

- 피드백은 자유입니다! 피드백을 남길 땐 확인하기 쉽도록 디스커션의 코멘트로 남겨주시면 됩니다.서로 글쓰기를 독려하는 문화를 만들어봅시다~*~*

- 글의 양식과 분량은 자유입니다! 그렇지만 여럿이서 참여하는 만큼, 정성을 들여서 작성해주세요😄

**Comment by shkisme** [comment]
꾸글쓰 해봅시다~

**Comment by stopmin** [comment]
앞으로 열심히 하겠습니닷!!
